src/main/scala/org/apache/mesos/chronos/scheduler/jobs/Containers.scala

```diff
@@ -9,14 +9,7 @@ object VolumeMode extends Enumeration {
   val RW, RO = Value
 }
 
-object NetworkMode extends Enumeration {
```

Comment:
Doing that, you would map a String to your custom Enum, which would then also have to store the user-network-name, which would then again be mapped to the `DockerInfo.Network.*` values in the MesosTaskBuilder.

What is the advantage of having the custom-enum?

If you remove the enum, the network just stays a String and the logic of mapping it to the Mesos API stays in the MesosTaskBuilder.